Library/Homebrew/api.rb

```diff
@@ -61,7 +59,7 @@ def fetch_json_api_file(endpoint, target:)
       begin
         begin
           args = curl_args.dup
-          args.prepend("--time-cond", target) if target.exist? && !target.empty?
+          args.prepend("--time-cond", target.to_s) if target.exist? && !target.empty?
```

Comment:
I believe it is safe to explicitly stringify the `Pathname` object here, but another option would be to define `args` as a mixed `Array` of `String` and `Pathname` objects.